RecoLocalCalo/EcalRecProducers/test/testEcalUncalibRechitProducerWithCC_cfg.py

```diff
@@ -0,0 +1,240 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+#from Configuration.ProcessModifiers.gpu_cff import gpu
+
+process = cms.Process('RECO', eras.Run2_2018)
+
+# import of standard configurations
```

Comment:
Better to remove all commented out lines for this test config if they are not needed.